docs/README.md

```diff
@@ -46,76 +47,66 @@ Samples in JavaScript, Python, and Java. Learn more at https://aka.ms/azai.
 
 # What is this sample?
 
-In this sample, we present **Contoso Outdoors**, a conceptual store specializing in outdoor gear for hiking and camping enthusiasts. This virtual store enhances customer engagement and sales support through an intelligent chat agent. This agent is powered by the **Retrieval Augmented Generation (RAG)** pattern within the **Microsoft Azure AI Stack**, enriched with **Semantic Kernel** and **Prompty** support.
+In this sample, we present **Contoso Outdoors**, a conceptual store specializing in outdoor gear for hiking and camping enthusiasts. This virtual store enhances customer engagement and sales support through an intelligent chat agent. Artificial Intelligence is used to integrate into the customer service experience, offering responses that are not only relevant but also personalized, drawing from an extensive product catalog and individual customer purchase histories.
 
-Artificial Intelligence integrates into the customer service experience, offering responses that are not only relevant but also personalized, drawing from the extensive product catalog and individual customer purchase histories.
+The sample uses the following Azure technologies:
+- [Azure AI Search](https://learn.microsoft.com/azure/search/) to create and manage search indexes for product catalog data.
+- [Azure Cosmos DB](https://learn.microsoft.com/azure/cosmos-db/) to store and manage customer purchase history data.
+- [Azure OpenAI](https://learn.microsoft.com/azure/ai-services/openai/) to deploy and manage key models for our copilot workflow.
+    - `text-embeddings-ada-002` for vectorizing user queries.
+    - `gpt-35-turbo` for generating chat responses.
 
-For our web application, we are using **.NET Stack**, and **AZD**, for an easy and quick deploy.
+It exposes a REST endpoint that can be called to interact with the chat agent and is implemented using:
+- [Semantic Kernel](https://learn.microsoft.com/en-us/semantic-kernel/overview/?tabs=Csharp) to access AI models, work with data stores, integrate prompts, and evaluate prompt/LLM performance.
+- **Prompty** to simplify prompt creation & iteration for this copilot scenario.
+- **ASP.NET Core WebAPI** to expose REST endpoints.
 
-This sample uses the [Azure AI](https://azure.microsoft.com/solutions/ai/). It leverages **Azure OpenAI** to our chat features and **Semantic Kernel** to manage and insert the prompt into our code, and to evaluate prompt/LLM performance.
-
-**Contoso Chat .NET** shows you how to:
+By exploring and deploying this sample, you will learn to:
 
 1. Build a retail copilot application using the **RAG pattern**.
-2. Ideate & iterate on application using [**Semantic Kernel**](https://learn.microsoft.com/en-us/semantic-kernel/overview/?tabs=Csharp) and **Prompty**.
+2. Ideate & iterate on application using [**Semantic Kernel**]() and **Prompty**.
```

Comment:
Is there a different link that should be added here? I see the previous one was deleted.